docs/app/documentation/component-docs/checkbox/examples/checkbox-form-group-example.component.html

```diff
@@ -23,7 +24,38 @@
         </div>
     </div>
 </form>
-<br><br>
+<br>
 Option 1 Value: {{ customForm.controls.firstOption.value }} <br>
 Option 2 Value: {{ customForm.controls.secondOption.value }} <br>
-Option 3 Value: {{ customForm.controls.thirdOption.value }}
+Option 3 Value: {{ customForm.controls.thirdOption.value }} <br>
+<br>
```

Comment:
I think the example gets too long. Instead, we can combine 1 disabled checkbox within the same group.